components/gc-featured-link/_base.scss

```
@@ -1,17 +1,20 @@
 /* Provisional featured link component */
 
-.provisional.gc-featured-link {
+.gc-featured-link {
 	background-color: $bg-gctheme;
```

Everything seems perfect, except after looking at the beta template for ILP, I noticed that the font-family should be Lato, @Ricokola can you add this line please?
```suggestion
	background-color: $bg-gctheme;
	font-family: Lato, sans-serif;
```